.travis.yml

```diff
@@ -11,14 +11,14 @@ scala:
   - 2.11.12
   - 2.12.8
   - 2.12.9
-  - 2.12.10
+  - 2.12.11
```

Comment:
You have to keep existing scala `2.12.10` version.
As it's a scalac plugin, it has to be cross-compiled to the exact versions of the compiler